Sources/Datadog/Datadog.swift

```diff
@@ -142,6 +142,18 @@ public class Datadog {
         )
     }
 
+    /// Add properties to the current user information
+    ///
+    /// There properties will be added to logs, traces and RUM events automatically.
+    /// - Parameters:
+    ///   - properties:
+    public static func addUserProperties(
```

Comment:
We're adding a new terminology here with the wording "UserProperties". 
Why not maintain the existing one, "ExtraInfo"?